src/conversion/buildviewconverter.js

```diff
@@ -141,11 +141,21 @@ class ViewConverterBuilder {
 	 * @param {String|RegExp} [value] View attribute value.
 	 * @returns {module:engine/conversion/buildviewconverter~ViewConverterBuilder}
 	 */
-	fromAttribute( key, value = /.*/ ) {
+	fromAttribute( key, value = /.+/ ) {
 		let pattern = {};
-		pattern[ key ] = value;
 
-		return this.from( pattern );
+		if ( key === 'style' || key === 'class' ) {
+			pattern[ key ] = value;
+		} else {
+			pattern.attribute = {};
+			pattern.attribute[ key ] = value;
+		}
+
+		this.from( pattern );
+
+		this._from[ this._from.length - 1 ].attributeKey = key;
```

Comment:
OK, I see where it's used (I must have made a typo looking for it before). Still, this is ugly, especially that you pass that key to `from()` anyway.